packages/elements/src/Menu/utils/dicKeyForDevices.ts

```diff
@@ -0,0 +1,19 @@
+import { capByPrefix } from "utils/src/text/capByPrefix";
+import { toCamelCase } from "utils/src/text/toCamelCase";
+
+/**
+ * Adds desktop,mobile and tablet keys to dictionary
+ */
+
+export const dicKeyForDevices = (key: string, value: string | number) => {
+  const keys = [key, capByPrefix("mMenu", key), capByPrefix("subMenu", key)];
+
+  return keys.reduce((menu, currentKey) => {
+    Object.assign(menu, {
+      [toCamelCase(currentKey)]: value,
+      [toCamelCase(capByPrefix("mobile", currentKey))]: value,
+      [toCamelCase(capByPrefix("tablet", currentKey))]: value
+    });
+    return menu;
+  }, {});
+};
```

Comment:
will be conflict with extracting subMenu keys 
![image](https://github.com/bagrinsergiu/MB-migration/assets/18303258/9095584d-1753-4ac8-b6ef-fd483c969fd6)